### ISSUE TITLE
Point to new location for input orography data

### ISSUE THREAD

**Issue by GeorgeGayno-NOAA** [NOAA-EMC/UFS_UTILS#757]
Files currently in the `./orog` directory will be moved to a new directory - `./raw/orog`. The files are:

- geo_em.d01.lat-lon.2.5m.HGT_M.nc
- GlobalLakeDepth.dat 
- GlobalLakeStatus.dat 
- gmted2010.30sec.int
- HGT.Beljaars_filtered.lat-lon.30s_res.nc
- landcover30.fixed
- thirty.second.antarctic.new.bin

Update any scripts to point to this location.

**Comment by GeorgeGayno-NOAA** [comment]
Will need to coordinate with the workflow group - @KateFriedman-NOAA @aerorahul @WalterKolczynski-NOAA

**Comment by KateFriedman-NOAA** [comment]
I've rsync'd the raw orog file movement into the other fix sets on supported platforms (Hera, Orion, Jet, WCOSS2) and setup temporary symlinks:

```
[glopara@hfe01 fix]$ pwd
/scratch1/NCEPDEV/global/glopara/fix
[glopara@hfe01 fix]$ ll orog/20220805/
total 64
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C1152
drwxr-sr-x 3 glopara global 4096 Jan  5 15:59 C1152.mx025
lrwxrwxrwx 1 glopara global   11 Jan  5 15:59 C1152.mx025_frac -> C1152.mx025
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C128
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C192
drwxr-sr-x 3 glopara global 4096 Jan  5 16:00 C192.mx050
lrwxrwxrwx 1 glopara global   10 Jan  5 16:00 C192.mx050_frac -> C192.mx050
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C3072
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C3359
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C3445
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C384
drwxr-sr-x 3 glopara global 4096 Jan  5 16:01 C384.mx025
lrwxrwxrwx 1 glopara global   10 Jan  5 16:01 C384.mx025_frac -> C384.mx025
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C48
drwxr-sr-x 3 glopara global 4096 Jan  5 16:02 C48.mx500
lrwxrwxrwx 1 glopara global    9 Jan  5 16:02 C48.mx500_frac -> C48.mx500
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C768
drwxr-sr-x 3 glopara global 4096 Jan  5 16:03 C768.mx025
lrwxrwxrwx 1 glopara global   10 Jan  5 16:03 C768.mx025_frac -> C768.mx025
drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C96
drwxr-sr-x 3 glopara global 4096 Jan  5 15:48 C96.mx100
lrwxrwxrwx 1 glopara global    9 Jan  5 15:48 C96.mx100_frac -> C96.mx100
lrwxrwxrwx 1 glopara global   34 Jan 18 15:51 GlobalLakeDepth.dat -> ../../raw/orog/GlobalLakeDepth.dat
lrwxrwxrwx 1 glopara global   35 Jan 18 15:51 GlobalLakeStatus.dat -> ../../raw/orog/GlobalLakeStatus.dat
lrwxrwxrwx 1 glopara global   55 Jan 18 15:51 HGT.Beljaars_filtered.lat-lon.30s_res.nc -> ../../raw/orog/HGT.Beljaars_filtered.lat-lon.30s_res.nc
lrwxrwxrwx 1 glopara global   47 Jan 18 15:50 geo_em.d01.lat-lon.2.5m.HGT_M.nc -> ../../raw/orog/geo_em.d01.lat-lon.2.5m.HGT_M.nc
lrwxrwxrwx 1 glopara global   34 Jan 18 15:51 gmted2010.30sec.int -> ../../raw/orog/gmted2010.30sec.int
lrwxrwxrwx 1 glopara global   32 Jan 18 15:51 landcover30.fixed -> ../../raw/orog/landcover30.fixed
lrwxrwxrwx 1 glopara global   46 Jan 18 15:51 thirty.second.antarctic.new.bin -> ../../raw/orog/thirty.second.antarctic.new.bin
[glopara@hfe01 fix]$ ll r
raw/      reg2grb2/ 
[glopara@hfe01 fix]$ ll raw/orog/
total 31028328
-rw-r--r-- 1 glopara global  1866240000 May 31  2018 GlobalLakeDepth.dat
-rw-r--r-- 1 glopara global        2163 May 31  2018 GlobalLakeDepth.txt
-rw-r--r-- 1 glopara global  1866240000 Jul 11  2022 GlobalLakeDepth_GLDBv3release.dat
-rw-r--r-- 1 glopara global  1866240000 Jul 11  2022 GlobalLakeDepth_GLOBathy.dat
-rw-r--r-- 1 glopara global   933120000 May 31  2018 GlobalLakeStatus.dat
-rw-r--r-- 1 glopara global        2805 May 31  2018 GlobalLakeStatus.txt
-rw-r--r-- 1 glopara global   933120000 Jul 11  2022 GlobalLakeStatus_GLDBv3release.dat
-rw-r--r-- 1 glopara global   933120000 Jul 11  2022 GlobalLakeStatus_MODISp.dat
-rw-r--r-- 1 glopara global   933120000 Jul 11  2022 GlobalLakeStatus_VIIRS.dat
-rw-r--r-- 1 glopara global 11197441020 Jan 14  2021 HGT.Beljaars_filtered.lat-lon.30s_res.nc
-rw-r--r-- 1 glopara global    14762250 Mar 11  2017 TOP8M_avg.20I4.asc
-rw-r--r-- 1 glopara global    14762250 Mar 11  2017 TOP8M_max.20I4.asc
-rw-r--r-- 1 glopara global     3690563 Mar 11  2017 TOP8M_slm.80I1.asc
-rw-r--r-- 1 glopara global      616356 Mar 11  2017 clmgrb
-rw-r--r-- 1 glopara global        5538 Mar 11  2017 clmgrb.index
-rw-r--r-- 1 glopara global        6836 Aug 14  2017 convert.f90
-rw-r--r-- 1 glopara global   447900364 Feb  6  2021 geo_em.d01.lat-lon.2.5m.HGT_M.nc
-rw-r--r-- 1 glopara global  3608064000 Feb 18  2014 gmted2010.30sec.flt
-rw-r--r-- 1 glopara global         263 May  5  2014 gmted2010.30sec.flt.ctl
-rw-r--r-- 1 glopara global  1866240000 Aug 14  2017 gmted2010.30sec.int
-rw-r--r-- 1 glopara global         264 May  5  2014 gmted2010.30sec.int.ctl
-rw-r--r-- 1 glopara global  1866240008 Mar 11  2017 gtopo30_gg.fine
-rw-r--r-- 1 glopara global  1866240000 Aug 16  2017 gtopo30_gg.fine.nh
-rw-r--r-- 1 glopara global   933120000 Mar 11  2017 landcover30.fixed
-rw-r--r-- 1 glopara global         240 May  5  2014 makefile
-rw-r--r-- 1 glopara global         313 Aug 14  2017 run.lsf
-rw-r--r-- 1 glopara global   622267212 Mar 11  2017 thirty.second.antarctic.new.bin
```

Let me know when I can remove (if at all) the symlinks. Thanks!

**Comment by GeorgeGayno-NOAA** [comment]
> I've rsync'd the raw orog file movement into the other fix sets on supported platforms (Hera, Orion, Jet, WCOSS2) and setup temporary symlinks:
> 
> ```
> [glopara@hfe01 fix]$ pwd
> /scratch1/NCEPDEV/global/glopara/fix
> [glopara@hfe01 fix]$ ll orog/20220805/
> total 64
> drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C1152
> drwxr-sr-x 3 glopara global 4096 Jan  5 15:59 C1152.mx025
> lrwxrwxrwx 1 glopara global   11 Jan  5 15:59 C1152.mx025_frac -> C1152.mx025
> drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C128
> drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C192
> drwxr-sr-x 3 glopara global 4096 Jan  5 16:00 C192.mx050
> lrwxrwxrwx 1 glopara global   10 Jan  5 16:00 C192.mx050_frac -> C192.mx050
> drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C3072
> drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C3359
> drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C3445
> drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C384
> drwxr-sr-x 3 glopara global 4096 Jan  5 16:01 C384.mx025
> lrwxrwxrwx 1 glopara global   10 Jan  5 16:01 C384.mx025_frac -> C384.mx025
> drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C48
> drwxr-sr-x 3 glopara global 4096 Jan  5 16:02 C48.mx500
> lrwxrwxrwx 1 glopara global    9 Jan  5 16:02 C48.mx500_frac -> C48.mx500
> drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C768
> drwxr-sr-x 3 glopara global 4096 Jan  5 16:03 C768.mx025
> lrwxrwxrwx 1 glopara global   10 Jan  5 16:03 C768.mx025_frac -> C768.mx025
> drwxr-sr-x 3 glopara global 4096 Aug  4 17:11 C96
> drwxr-sr-x 3 glopara global 4096 Jan  5 15:48 C96.mx100
> lrwxrwxrwx 1 glopara global    9 Jan  5 15:48 C96.mx100_frac -> C96.mx100
> lrwxrwxrwx 1 glopara global   34 Jan 18 15:51 GlobalLakeDepth.dat -> ../../raw/orog/GlobalLakeDepth.dat
> lrwxrwxrwx 1 glopara global   35 Jan 18 15:51 GlobalLakeStatus.dat -> ../../raw/orog/GlobalLakeStatus.dat
> lrwxrwxrwx 1 glopara global   55 Jan 18 15:51 HGT.Beljaars_filtered.lat-lon.30s_res.nc -> ../../raw/orog/HGT.Beljaars_filtered.lat-lon.30s_res.nc
> lrwxrwxrwx 1 glopara global   47 Jan 18 15:50 geo_em.d01.lat-lon.2.5m.HGT_M.nc -> ../../raw/orog/geo_em.d01.lat-lon.2.5m.HGT_M.nc
> lrwxrwxrwx 1 glopara global   34 Jan 18 15:51 gmted2010.30sec.int -> ../../raw/orog/gmted2010.30sec.int
> lrwxrwxrwx 1 glopara global   32 Jan 18 15:51 landcover30.fixed -> ../../raw/orog/landcover30.fixed
> lrwxrwxrwx 1 glopara global   46 Jan 18 15:51 thirty.second.antarctic.new.bin -> ../../raw/orog/thirty.second.antarctic.new.bin
> [glopara@hfe01 fix]$ ll r
> raw/      reg2grb2/ 
> [glopara@hfe01 fix]$ ll raw/orog/
> total 31028328
> -rw-r--r-- 1 glopara global  1866240000 May 31  2018 GlobalLakeDepth.dat
> -rw-r--r-- 1 glopara global        2163 May 31  2018 GlobalLakeDepth.txt
> -rw-r--r-- 1 glopara global  1866240000 Jul 11  2022 GlobalLakeDepth_GLDBv3release.dat
> -rw-r--r-- 1 glopara global  1866240000 Jul 11  2022 GlobalLakeDepth_GLOBathy.dat
> -rw-r--r-- 1 glopara global   933120000 May 31  2018 GlobalLakeStatus.dat
> -rw-r--r-- 1 glopara global        2805 May 31  2018 GlobalLakeStatus.txt
> -rw-r--r-- 1 glopara global   933120000 Jul 11  2022 GlobalLakeStatus_GLDBv3release.dat
> -rw-r--r-- 1 glopara global   933120000 Jul 11  2022 GlobalLakeStatus_MODISp.dat
> -rw-r--r-- 1 glopara global   933120000 Jul 11  2022 GlobalLakeStatus_VIIRS.dat
> -rw-r--r-- 1 glopara global 11197441020 Jan 14  2021 HGT.Beljaars_filtered.lat-lon.30s_res.nc
> -rw-r--r-- 1 glopara global    14762250 Mar 11  2017 TOP8M_avg.20I4.asc
> -rw-r--r-- 1 glopara global    14762250 Mar 11  2017 TOP8M_max.20I4.asc
> -rw-r--r-- 1 glopara global     3690563 Mar 11  2017 TOP8M_slm.80I1.asc
> -rw-r--r-- 1 glopara global      616356 Mar 11  2017 clmgrb
> -rw-r--r-- 1 glopara global        5538 Mar 11  2017 clmgrb.index
> -rw-r--r-- 1 glopara global        6836 Aug 14  2017 convert.f90
> -rw-r--r-- 1 glopara global   447900364 Feb  6  2021 geo_em.d01.lat-lon.2.5m.HGT_M.nc
> -rw-r--r-- 1 glopara global  3608064000 Feb 18  2014 gmted2010.30sec.flt
> -rw-r--r-- 1 glopara global         263 May  5  2014 gmted2010.30sec.flt.ctl
> -rw-r--r-- 1 glopara global  1866240000 Aug 14  2017 gmted2010.30sec.int
> -rw-r--r-- 1 glopara global         264 May  5  2014 gmted2010.30sec.int.ctl
> -rw-r--r-- 1 glopara global  1866240008 Mar 11  2017 gtopo30_gg.fine
> -rw-r--r-- 1 glopara global  1866240000 Aug 16  2017 gtopo30_gg.fine.nh
> -rw-r--r-- 1 glopara global   933120000 Mar 11  2017 landcover30.fixed
> -rw-r--r-- 1 glopara global         240 May  5  2014 makefile
> -rw-r--r-- 1 glopara global         313 Aug 14  2017 run.lsf
> -rw-r--r-- 1 glopara global   622267212 Mar 11  2017 thirty.second.antarctic.new.bin
> ```
> 
> Let me know when I can remove (if at all) the symlinks. Thanks!

Great. I will submit a PR soon.
